src/main/java/spoon/support/compiler/jdt/JDTConstants.java

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (C) 2006-2019 INRIA and contributors
+ *
+ * Spoon is available either under the terms of the MIT License (see LICENSE-MIT.txt) of the Cecill-C License (see LICENSE-CECILL-C.txt). You as the user are entitled to choose the terms under which to adopt Spoon.
+ */
+package spoon.support.compiler.jdt;
+
+/**
+ * Extracts a subset of constants as defined in {@link org.eclipse.jdt.internal.compiler.lookup.TypeConstants}, since
+ * this class is only meant for internal use.
+ */
+public class JDTConstants {
+
+	private JDTConstants() {}
```

Comment:
Still some complains from checkstyle:
> ERROR] /home/travis/build/INRIA/spoon/src/main/java/spoon/support/compiler/jdt/JDTConstants.java:14:33: '{' is not followed by whitespace. [WhitespaceAround]
[ERROR] /home/travis/build/INRIA/spoon/src/main/java/spoon/support/compiler/jdt/JDTConstants.java:14:33: '}' is not preceded with whitespace. [WhitespaceAround]